module/Activity/view/activity/activity/create.phtml

```diff
@@ -5,9 +5,10 @@ $this->headLink()
 $this->headScript()
     ->prependFile($this->basePath() . '/js/activity-create.js')
     ->prependFile($this->basePath() . '/js/jquery.datetimepicker.full.min.js');
-$form->prepare();
+if (!$success) {$form->prepare();}
 ?>
 <div class="container">
+    <?php if (!$success): ?>
```

Comment:
This is a if else structure that now spans more than 200 lines of code. I think it would be better to split it into more than one template. Either having a succes and a non success template, or doing this:
```
if (a):
    loadTemplate('notsuccess')
else:
    loadTemplate('succes')
endif;
```

Otherwise the templates become very incomprehensible quite fast